lib/cog/relay/tracker.ex

```diff
@@ -120,15 +120,15 @@ defmodule Cog.Relay.Tracker do
   end
 
   @doc """
-  Return a list of relays serving the specified bundle version. If the bundle is
-  disabled, return an empty list.
+  Return a tuple with a list of ALL relays serving the bundle and a list of the
+  ENABLED relays serving the bundle.
   """
-  @spec relays(t, bundle_name, version) :: [String.t]
+  @spec relays(t, bundle_name, version) :: {[String.t], [String.t]}
   def relays(tracker, bundle_name, bundle_version) when is_binary(bundle_name) do
-    tracker.map
-    |> Map.get({bundle_name, bundle_version}, MapSet.new)
-    |> MapSet.difference(tracker.disabled)
-    |> MapSet.to_list
+    relays = Map.get(tracker.map, {bundle_name, bundle_version}, MapSet.new)
+    enabled_relays = MapSet.difference(relays, tracker.disabled)
+
+    {MapSet.to_list(relays), MapSet.to_list(enabled_relays)}
```

Comment:
I think returning descriptive atoms instead of sets of relays would be more intention revealing.

`{[], []}` becomes `:no_relays`
`{[...], []}` becomes `:no_enabled_relays`
`{[...], [...]}` becomes `[...]`

where `[...`] indicates a non-empty list.